Make multi platform targets parse deployment target per platform. (#510)
yonaskolb/XcodeGen: CHANGELOG.md

#### Added
- Added `missingConfigFiles` to `options.disabledValidations` to optionally skip checking for the existence of config files.
- Added ability to automatically include Carthage related dependencies via `includeRelated: true` [#506](https://github.com/yonaskolb/XcodeGen/pull/506) @rpassis

> - Added ability to customize deployment target for Multi Platform targets. [#510](https://github.com/yonaskolb/XcodeGen/pull/510) @ainopara

Reviewer: ```suggestion
- Added ability to define a per-platform `deploymentTarget` for Multi-Platform targets. [#510](https://github.com/yonaskolb/XcodeGen/pull/510) @ainopara
```
